newIDE/app/src/GameDashboard/GameDetailsDialog.js

```diff
@@ -184,27 +192,36 @@ export const GameDetailsDialog = ({
           author: authorAcls,
         });
       } catch (error) {
-        console.error('Unable to update the game owners or authors:', error);
+        console.error(
+          'Unable to update the game owners or authors:',
+          error.response || error.message
+        );
         showErrorBox({
           message:
             i18n._(t`Unable to update the game owners or authors.`) +
             ' ' +
-            i18n._(t`Verify your internet connection or try again later.`),
+            ((error.response && error.response.data) ||
```

Comment:
Usually we avoid surfacing the response data directly to the user - in case we want more info we would ask the backend to return an error with a body like `{ code: 'area/what-happened', }`, so that the UI can react accordingly (or display a generic message if the code is not present/not expected).